src/BizHawk.Emulation.Cores/Waterbox/WaterboxMemoryDomain.cs

```diff
@@ -140,7 +140,8 @@ public StubResolver(IntPtr p)
 
 		public static MemoryDomainAccessStub Create(IntPtr p, IMonitor monitor)
 		{
-			return BizInvoker.GetInvoker<MemoryDomainAccessStub>(new StubResolver(p), monitor, CallingConventionAdapters.Waterbox);
+			return BizInvoker.GetInvoker<MemoryDomainAccessStub>(

```

Comment:
This has the same issue as nyma frame procs, but shouldn't be that way.  `WaterboxDepartureOnly` is probably a super broken loaded footgun, revisit